package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/aio-cli-plugin-app-templates",
-  "version": "1.0.0-beta.11",
+  "version": "1.0.0-beta.12",
```

Comment:
I think we can make a first release, `1.0.0` 👍

Reply:
Done ✅